src/lib/cloudinary.js

```diff
@@ -224,7 +224,32 @@ async function updateHtmlImagesToCloudinary(html, options = {}) {
       }
     }
 
-    $img.setAttribute('src', cloudinaryUrl)
+    $img.setAttribute('src', cloudinaryUrl);
+     // convert srcset images to cloudinary
+    const srcset = $img.getAttribute('srcset');
+    if (srcset) {
+      // convert all srcset urls to cloudinary urls using getCloudinaryUrl function in a Promise.all
+      const srcsetUrls = srcset.split(',').map((url) => url.trim().split(' '));
+      const srcsetUrlsPromises = srcsetUrls.map((url) =>{ 
+        const exists = asset(url[0]);
+        if ( exists && deliveryType === 'upload' ) {
+          return exists.cloudinaryUrl;
+        }
+        return getCloudinaryUrl({
+        deliveryType,
+        folder,
+        path: url[0],
+        localDir,
+        uploadPreset,
+        remoteHost
```

Comment:
if you can make sure the object parameters are intended here thatd be great

Reply:
Sure that's good.